tests/rptest/tests/self_test_test.py

```diff
@@ -70,8 +70,8 @@ def assert_fail(report, error_msg):
             assert 'error' in report
             assert report['error'] == error_msg
 
-        read_tests = ['list', 'download']
-        write_tests = ['upload', 'delete']
+        read_tests = ['List', 'Head', 'Get']
+        write_tests = ['Put', 'Delete', 'Plural Delete']
```

Comment:
nit: generally it's a good practice to ensure individual points in the git history pass tests. This should probably be squashed with the previous commit.

Reply:
Squashed commit.